AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -495,6 +495,7 @@ experts-exchange.com##.adBanner-comment
 experts-exchange.com##.ctaSolutionStickyWrapper
 !
 vchehle.ua##.promo-popup
+udf.by##a[href^="https://ok.ru/group/"][href*="longbanner"]
```

Comment:
Exchange the rule to `udf.by##.category_news.last` and move to `annoyances` section with comment above.